modules/playback.js

```diff
@@ -1,5 +1,9 @@
 import * as MAESTRO from "./config.js";
 
+class PlaybackSettings{
```

Comment:
Instead of a static inner class, I'd prefer adding a property to the `game.maestro` object. Eg. `game.maestro.pausedTracks` or similar.